db/src/main/java/service/DBService_salary.java

```diff
@@ -0,0 +1,53 @@
+package service;
+
+import db.DB;
+import dto.Employee_salary;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class DBService_salary {
```

Comment:
Class name does not follow Java Naming Conventions.
It should be `DBServiceSalary`